lib/router.js

```diff
@@ -109,6 +109,10 @@ module.exports = class Router extends express.Router {
       });
       req.next();
     });
+
+    // Enable body parsing for all routes
+    this.use(express.json());
```

Comment:
This is already in silex in my case, last time we discussed this I think we decided to let the main application configure express?

Reply:
Forgot that! But I'll move it to `index.js` to make CE usable as a standalone. 

Also, shouldn't we use null checks when using body in the router? Because if someone forgot to enable body parser, we crash.